packages/block-editor/src/components/inserter/quick-inserter.js

```diff
@@ -70,13 +70,10 @@ export default function QuickInserter( {
 		}
 	}, [ setInserterIsOpened ] );
 
-	const { __unstableSetInsertionPoint } = useDispatch( blockEditorStore );
-
 	// When clicking Browse All select the appropriate block so as
 	// the insertion point can work as expected
 	const onBrowseAll = () => {
-		__unstableSetInsertionPoint( rootClientId, blockIndex );
-		setInserterIsOpened( true );
+		setInserterIsOpened( { rootClientId, blockIndex } );
```

Comment:
Worth noting that this is an experimental API (`getSettings().__experimentalSetIsInserterOpened`).